docs/LV_client/utilities/mono_build_name.md

```diff
@@ -0,0 +1,39 @@
+# mono_build_name.vi
+
+<p class="img_container">
+<img class="lg_img" src="../mono_build_name.png"/>
+</p>
+
+### Description
+
+Builds the name of a file based on the type of sensor and the custom name the user provided.
+
+For technical support contact us at <b>support@monodrive.io</b>  
+
+### Inputs
+
+- **Configuration:**  Configuration for the sensors. Each sensor has an output
+named "Configuration" wire that output to this input.
+ 
+
+- **Name:**  Name appended to the type of sensor. For example if the type
+of sensor is Camera and ther name is "Front" the resulting
```

Comment:
Need to change ther to "it's"and add comma after "Front"
of sensor is Camera and it's name is "Front", the resulting